ckks/params.go

```diff
@@ -285,17 +266,17 @@ var (
 // type (RingType) and the number of slots (in log_2, LogSlots). If left unset, standard default values for
 // these field are substituted at parameter creation (see NewParametersFromLiteral).
 type ParametersLiteral struct {
```

Comment:
The comment above says:
```
// ParametersLiteral is a literal representation of BFV parameters
```
Shouldn't it say _CKKS_ parameters instead? Sorry for mentioning this when it's not related to the current PR, but I will forget otherwise and it's probably worth fixing if it is a mistake.

Reply:
Yes this is a typo, that I also just noticed, thank you.